calendar-bundle/src/Resources/contao/languages/en/tl_calendar_events.xlf

```diff
@@ -56,6 +56,12 @@
       <trans-unit id="tl_calendar_events.pageTitle.1">
         <source>Here you can add a custom meta title to overwrite the default page title.</source>
       </trans-unit>
+      <trans-unit id="tl_calendar_events.robots.0">
+        <source>Robots tag</source>
+      </trans-unit>
+      <trans-unit id="tl_calendar_events.robots.1">
+        <source>Here you can override how search engines handle the page. If empty the page setting is used.</source>
```

Comment:
Should this be _event_ instead of _page_?

Reply:
Resolved in e4056844494cffdfa1059e19d213deb6250ac083